Atomic/syscontainers.py

```diff
@@ -149,6 +162,16 @@ def install_system_container(self, image, name):
         if not repo:
             raise ValueError("Cannot find a configured OSTree repo")
 
+        if not self.user:
+            runc_found = True
+        try:
+            util.check_call([RUNC_PATH, "--version"], stdout=DEVNULL)
+        except util.FileNotFound:
+            runc_found = False
+
+        if not runc_found:
+            raise ValueError("Cannot install the container: runc is needed to run system containers")
+
```

Comment:
thanks, I have tested that if the exception is caught then Python3 doesn't error out, I have updated and rebased the patches.